{
  "channels": [
    "defaults",
    "https://repo.anaconda.com/pkgs/main"
  ],
  "conda_build_version": "3.13.0",
  "conda_private": false,
  "conda_version": "4.5.10",
  "env_vars": {
    "CIO_TEST": "<not set>"
  },
  "extra": {
    "copy_test_source_files": true,
    "final": true
  },
  "home": "https://github.com/ContinuumIO/navigator-updater",
  "license": "proprietary - Continuum Analytics, Inc.",
  "license_family": "Proprietary",
  "root_pkgs": [
    "asn1crypto 0.24.0 py36_0",
    "beautifulsoup4 4.6.3 py36_0",
    "ca-certificates 2018.03.07 0",
    "certifi 2018.8.13 py36_0",
    "cffi 1.11.5 py36h945400d_0",
    "chardet 3.0.4 py36h420ce6e_1",
    "conda 4.5.10 py36_0",
    "conda-build 3.13.0 py36_0",
    "conda-env 2.6.0 h36134e3_1",
    "console_shortcut 0.1.1 h6bb2dd7_3",
    "cryptography 2.2.2 py36hfa6e2cd_0",
    "filelock 3.0.4 py36_0",
    "git 2.18.0 h6bb4b03_0",
    "glob2 0.6 py36_0",
    "idna 2.6 py36h148d497_1",
    "jinja2 2.10 py36_0",
    "m2-msys2-runtime 2.5.0.17080.65c939c 3",
    "m2-patch 2.7.5 2",
    "markupsafe 1.0 py36hfa6e2cd_1",
    "menuinst 1.4.14 py36hfa6e2cd_0",
    "msys2-conda-epoch 20160418 1",
    "openssl 1.0.2p hfa6e2cd_0",
    "pip 10.0.1 py36_0",
    "pkginfo 1.4.2 py36_1",
    "psutil 5.4.7 py36hfa6e2cd_0",
    "pycosat 0.6.3 py36h413d8a4_0",
    "pycparser 2.18 py36hd053e01_1",
    "pyopenssl 18.0.0 py36_0",
    "pysocks 1.6.8 py36_0",
    "python 3.6.5 h0c2934d_0",
    "pywin32 223 py36hfa6e2cd_1",
    "pyyaml 3.13 py36hfa6e2cd_0",
    "requests 2.18.4 py36h4371aae_1",
    "ruamel_yaml 0.15.37 py36hfa6e2cd_2",
    "setuptools 39.2.0 py36_0",
    "six 1.11.0 py36h4db2310_1",
    "urllib3 1.22 py36h276f60a_0",
    "vc 14.1 h0510ff6_3",
    "vs2015_runtime 15.5.2 3",
    "wheel 0.31.1 py36_0",
    "wincertstore 0.2 py36h7fe50ca_0",
    "win_inet_pton 1.0.1 py36he67d7fd_1",
    "yaml 0.1.7 hc54c509_2"
  ],
  "summary": "Anaconda Navigator Updater"
}